src/java.desktop/macosx/native/libawt_lwawt/awt/CTrayIcon.m

```diff
@@ -83,17 +99,23 @@ -(void) dealloc {
     // the item's view to nil: it can lead to a crash in some scenarios.
```

Comment:
This comment seems outdated, we either need to nullify the menu delegate or follow the old approach and update the comment on why the menuDelegate is not set to nil.